docs/routing/APP_SERVER_ROUTING.md

```diff
@@ -21,6 +24,7 @@ export class AppModule {}
 ```
 
 By default, the ``APP_BASE_HREF`` will be provided using one of the following items (by priority):
-  * Disabled if it's not the PROD environment
+  * Disabled if it's not the PROD environment (`environment.ENVIRONMENT` === 'prod')
   * the content of ``data-appbasehref`` attribute of ``body``
+  * the value of ``environment.APP_BASE_HREF``
```

Comment:
might be in a lesser priority than the --app-base-href? (if this is still valid)